lib/install/management/dispatcher.go

```diff
@@ -181,8 +196,8 @@ func (d *Dispatcher) Dispatch(conf *metadata.VirtualContainerHostConfigSpec, set
 		return err
 	}
 
-	if err = d.removeApplianceIfForced(conf); err != nil {
-		return errors.Errorf("%s", err)
+	if err = d.checkExistence(conf); err != nil {
```

Comment:
As we plan to separate the modules to different services in the future, I assume dispatcher will be the service name, and it provides different API for create, delete, etc.
But yeah, single purpose function will be more clear. Will do it in next PR.